cranelift/isle/docs/language-reference.md

```diff
@@ -1438,22 +1438,33 @@ newline). The grammar accepted by the parser is as follows:
 
 <ISLE> ::= <def>*
 
-<def> ::= "(" "type" <typedecl> ")"
+<def> ::= "(" "pragma" <pragma> ")"
+        | "(" "type" <typedecl> ")"
         | "(" "decl" <decl> ")"
+        | "(" "spec" <spec> ")"
+        | "(" "model" <model> ")"
+        | "(" "form" <form> ")"
+        | "(" "instantiate" <instantiation> ")"
```

Comment:
Can we put these four in a separate `<veri-def>` production? In general we haven't documented the verification extensions to ISLE as they're somewhat still in flux; I want to try to keep them separate so as not to confuse users getting started.